examples/runc-hook/deploy.yaml

```diff
@@ -0,0 +1,140 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: runc-hook
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: runc-hook
+  namespace: runc-hook
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: runc-hook-cluster-role
+rules:
+- apiGroups: [""]
+  resources: ["pods"]
+  verbs: ["get", "watch", "list"]
+- apiGroups: ["security.openshift.io"]
+  # It is necessary to use the 'privileged' security context constraints to be
+  # able mount host directories as volumes, use the host networking, among others.
+  # This will be used only when running on OpenShift:
+  # https://docs.openshift.com/container-platform/4.9/authentication/managing-security-context-constraints.html#default-sccs_configuring-internal-oauth
+  resources: ["securitycontextconstraints"]
+  resourceNames: ["privileged"]
+  verbs: ["use"]
+---
+kind: ClusterRoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: runc-hook-cluster-role-binding
+subjects:
+- kind: ServiceAccount
+  name: runc-hook
+  namespace: runc-hook
+roleRef:
+  kind: ClusterRole
+  name: runc-hook-cluster-role
+  apiGroup: rbac.authorization.k8s.io
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: hook-config
+  namespace: runc-hook
+data:
+  prestart.sh: |
+    #!/bin/sh
+    (cat ; echo) >> /tmp/hooks.log
```

Comment:
shouldn't this read stdin or something?

Reply:
The `cat` command reads from stdin and write into the file. The command `echo` adds a new line, otherwise the json will be on the same line as other output.